src/persist/src/mem.rs

```diff
@@ -451,7 +451,7 @@ impl MemRegistry {
     }
 
     /// Returns a [RuntimeClient] with unreliable storage backed by the given
-    /// [`UnreliableHandle`].
+    /// [`UnreliableHandleOld`].
```

Comment:
super nit: these might be in the wrong commit 😅

Reply:
done!